draft-ietf-oauth-device-flow.xml

```diff
@@ -99,13 +97,16 @@
     <keyword>Smart Objects</keyword>
```

Comment:
Maybe we should add other key words, like

Internet of Things
Internet of Things Security
OAuth for Constrained Devices
OAuth IoT Security